test/cmdlineargs.jl

```diff
@@ -61,15 +61,6 @@ let exename = `$(Base.julia_cmd()) --precompiled=yes --startup-file=no`
     @test !success(`$exename -L`)
     @test !success(`$exename --load`)
 
-    # --cpu-target
-    # NOTE: this test only holds true if image_file is a shared library.
```

Comment:
No. It could be an CPU name that we don't recognize but llvm does so it shouldn't be blocked at this level and should be passed to LLVM. Unfortunately LLVM does not throw an error in that case and I'm not aware of any method to ask LLVM if it is a supported target name at runtime.